garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stool.lua

```diff
@@ -103,7 +103,7 @@ function ToolObj:Init() end
 
 function ToolObj:GetMode()		return self.Mode end
 function ToolObj:GetSWEP()		return self.SWEP end
-function ToolObj:GetOwner()		return self:GetSWEP().Owner or self.Owner end
+function ToolObj:GetOwner()		return self:GetSWEP():GetOwner() or self:GetOwner() end

```

Comment:
This looks like a bug. `self.Owner` in this case would be correct.

Reply:
That's right. Should `self:GetSwep():GetOwner()` ever return `nil`, this will cause a stack overflow.

```suggestion
function ToolObj:GetOwner()		return self:GetSWEP():GetOwner() or self.Owner end
```